### PR TITLE
Update CPU model to Mobilenet v2

### PR DESCRIPTION
**Pull request by mattheys**
Inference speed went from ~470ms to ~530ms, however average confidence went from ~75% to ~90%+

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -46,10 +46,8 @@ RUN apt -qq update && apt -qq install --no-install-recommends -y \
 # get model and labels
 RUN wget -q https://github.com/google-coral/edgetpu/raw/master/test_data/mobilenet_ssd_v2_coco_quant_postprocess_edgetpu.tflite -O /edgetpu_model.tflite --trust-server-names
 RUN wget -q https://dl.google.com/coral/canned_models/coco_labels.txt -O /labelmap.txt --trust-server-names
-RUN wget -q https://storage.googleapis.com/download.tensorflow.org/models/tflite/coco_ssd_mobilenet_v1_1.0_quant_2018_06_29.zip -O /cpu_model.zip && \
-    unzip /cpu_model.zip detect.tflite -d / && \
-    mv /detect.tflite /cpu_model.tflite && \
-    rm /cpu_model.zip
+RUN wget -q https://github.com/google-coral/edgetpu/raw/master/test_data/mobilenet_ssd_v2_coco_quant_postprocess.tflite -O /cpu_model.tflite
```
Comment (blakeblackshear) [inline Dockerfile:49]: This is the model optimized for the edgetpu that I am downloading above. I don't remember the it working in CPU mode. Even if it does, no reason to download twice. Is this really working?

Comment (mattheys) [reply Dockerfile:49]: No it's the cpu model as it's missing _edgetpu in the file name, they are two different files.

Comment (mattheys) [reply Dockerfile:49]: Oh and yes it is working, saw inference speed change from 470ms to 530ms and confidence go from 70% to 90%

Comment (blakeblackshear) [reply Dockerfile:49]: Totally overlooked that. Thanks.

**Pull request merged**